.travis.yml

```diff
@@ -21,10 +20,14 @@ env:
 
 matrix:
   fast_finish: true
+  include:
+    - php: 5.6
+      env: SYMFONY_VERSION="2.8.*"
```

Comment:
Elcodi uses components, and if you check some components (maybe all of them) composer.json files, they are compatible with 2.8 versions as well, so, and because GaufretteBundle don't allow Symfony3 in some components, they are not installed in Symfony3.

Can you check that please by adding explicitelly symfony/symfony@3.0.0 please?